server/src/main/java/org/opensearch/snapshots/RestoreService.java

```diff
@@ -629,6 +653,48 @@ private void checkAliasNameConflicts(Map<String, String> renamedIndices, Set<Str
                         }
                     }
 
+                    private String[] getIgnoreSettingsInternal() {
```

Comment:
IIUC, if the new cluster where snapshot is being restored is not remote store enabled, then we ignore all remote store related settings?

Reply:
yes, with this method, we are ignoring remote store related settings if the domain is not remote store enabled.